content/doc/book/blueocean/dashboard.adoc

```diff
@@ -19,105 +19,78 @@ endif::[]
 
 = Dashboard
 
-Blue Ocean's "Dashboard" is the default view shown when you open Blue Ocean and
-shows an overview of all Pipeline projects configured on a Jenkins instance.
+Blue Ocean's *Dashboard* is the default view shown when opening Blue Ocean.
+It disaplays an overview of all Pipeline projects configured on a Jenkins instance.
 
 include::doc/book/blueocean/_blue-ocean-status.adoc[]
 
-The Dashboard consists of a blue link:#navigation-bar[navigation bar] at the
-top, the link:#pipelines-list[Pipelines list], as well as the
-link:#favorites-list[Favorites list].
+The Dashboard consists of a blue link:#navigation-bar[navigation bar] at the top, the link:#pipelines-list[Pipelines list], and the link:#favorites-list[Favorites list].
 
 [.boxshadow]
-image:blueocean/dashboard/overview.png[Overview of the
-Dashboard,role=center,width=100%]
+image:blueocean/dashboard/overview.png[Overview of the Dashboard,role=center,width=100%]
 
 == Navigation bar
 
-The Dashboard includes the blue-colored
-link:../getting-started#navigation-bar[navigation bar] along the top of the
-interface.
+The Dashboard includes the blue link:../getting-started#navigation-bar[navigation bar] along the top of the interface.
```

Comment:
```suggestion
The user interface includes the blue link:../getting-started#navigation-bar[navigation bar] along the top of the interface.
```

Reply:
leaving dashboard terminology, discussed in slack